src/main/resources/mybatis/collections.xml

```diff
@@ -102,4 +102,48 @@
         ) as collection
     </select>
 
+
+    <select id="getStatisticalTimeSeriesJson" parameterType="String" resultType="String">
+        select row_to_json(fsts)
+          from ( select 'Feature' as "type",
```

Comment:
I am not a fan of using mixed case object names in sql. Each database handles them in their own way. I realize this is just aliases within a query, but copy paste runs rampant and could cause the uninformed much grief when a query does not work.

Reply:
Yes I get you. The case if forced because that is what ends up as the JSON key. I usually do not use much more that a char for them but in the special case of 'timeSeries' that is the key as it should be for the JSON response. Thus, since pg is generating the JSON I had to conform to its protocol.